community/server/src/docs/ops/security.asciidoc

```diff
@@ -51,6 +51,10 @@ The Neo4j server includes built in support for SSL encrypted communication over
 The first time the server starts, it automatically generates a self-signed SSL certificate and a private key.
 Because the certificate is self signed, it is not safe to rely on for production use, instead, you should provide your own key and certificate for the server to use.
 
+[CAUTION]
+Using autogeneration of self-signed SSL certificates will not work if the Neo4j server has been configured with multiple connectors that do not bind to the same IP address.
+If you need to use multiple IP addresses, please configure certificates manually and consider using multi-host or wildcard certificates instead.
+
```

Comment:
Small change in wording to increase clarity. This change needs to be done in the new manual repository. I have given you access. The file to modify in that repo is: src/main/operations-manual/security/security.asciidoc

Using auto-generation of self-signed SSL certificates will not work if the Neo4j server has been configured with multiple connectors that bind to different IP addresses. If you need to use multiple IP addresses, please configure certificates manually or consider using multi-host or wildcard certificates instead.